XML/TestCases/StressTests.xml

```diff
@@ -275,9 +275,26 @@
 			<param>maxIteration=MAX_CPU_OFFLINEONLINE</param>
 		</TestParameters>
 		<Platform>Azure</Platform>
-		<Category>Functional</Category>
+		<Category>Stress</Category>
 		<Area>CORE</Area>
```

Comment:
CORE or POWER?

Reply:
Can you re-check? This is correct. I fixed the old error. The line 278 is for CPU offline stress test, and was Functional. so I fixed it. CPU offline is CORE test.